PowerShellTools/ServiceManagement/PowershellHostProcessException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.ServiceManagement
+{
+    [Serializable]
+    public class PowershellHostProcessException : Exception

```

Comment:
I want this exception noticeable.

From: Eric Deng [mailto:notifications@github.com]
Sent: Wednesday, January 21, 2015 11:28 AM
To: Microsoft/poshtools
Cc: Huanhuan Sun
Subject: Re: [poshtools] Infrastructure for out-of-proc, includes initializing a process and creating a WCF channel for servicing (#36)


In PowerShellTools/ServiceManagement/PowershellHostProcessException.cs<https://github.com/Microsoft/poshtools/pull/36#discussion_r23326517>:

> @@ -0,0 +1,16 @@

> +﻿using System;

> +using System.Collections.Generic;

> +using System.Linq;

> +using System.Text;

> +using System.Threading.Tasks;

> +

> +namespace PowerShellTools.ServiceManagement

> +{

> +    [Serializable]

> +    public class PowershellHostProcessException : Exception

Why do we need this since we should be able to serialize Exception directly? Or you are thinking about to do specific action for its handling?

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/poshtools/pull/36/files#r23326517>.